logger/log.go

```diff
@@ -38,13 +38,37 @@ type Logger struct {
 	fl         *fileLogger
 }
 
-// NewStdLogger creates a logger with output directed to Stderr
-func NewStdLogger(time, debug, trace, colors, pid bool) *Logger {
+type LogOption interface {
+	isLoggerOption()
```

Comment:
Should this be upper case so it is also public?

Reply:
Another approach could be add new constructor?

NewStdUTCLogger() and NewFileUTCLogger() 

Just a thought, not suggesting either way is better or not.

Reply:
The interface is exported but the function to match isn't, that way it prevents other packages from defining new `LogOptions` that won't make sense to us (as they can't implement the `isLoggerOption` to do so).

Also I did think about adding new functions for UTC specifically, but figured an options-based approach would be more future-proof in case there are more options we want to add in the future.

Reply:
ok thanks.. Appreciate the feedback.